internal/commands/tunnel/tunnel.go

```diff
@@ -95,6 +132,29 @@ func (o *TunnelOptions) Complete(cmd *cobra.Command, args []string) error {
 		o.PublicKeyFile = fmt.Sprintf("%s/.ssh/id_rsa.pub", home)
 	}
 
+	if len(o.BastionHostID) == 0 && len(o.ForwardHost) != 0 {
+		return fmt.Errorf("cat't complete options: for forward-host should be specified bastion host id")
```

Comment:
Let's make it more user-oriented:
```suggestion
		return fmt.Errorf("cat't establish tunnel: --forward-host parameter requires --bastion-instance-id")
```